src/common/Pagination/index.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import {
+    Wrapper,
+    ArrowIcon,
+    Button,
+    TextButton,
```

Comment:
The name TextButton suggests a whole button with text, maybe call it ButtonText or just Text, which would suggest only text in the button. But this is my opinion 🙂

Reply:
Ok i will change it! Thankyou :)